woocommerce/woocommerce-ios
# Refund Details: refund details and reason note sections

Closes #1697 

⚠️ Change base to `develop` before merging.

This PR completes the Refund Details screen. 

For reference, here is the full refund details page:
<img src="https://user-images.githubusercontent.com/1062444/71692247-f6e3d400-2d6e-11ea-9152-dbd149d2b955.png" width="350" />

## To Test
1. On the website, choose an order and create a refund by item. Refund 2 items or more (but not a full refund, it needs to be a partial refund) and add a note.
2. After creating the refund, go to the app and navigate to the Order Details for the chosen order.
3. The "Refunded" cell should be the new semantic color, "link", which is an active (pink) color. Tap on the "Refunded" cell.
4. The Refund Details screen should appear. The new Refund Details section should show. If a note was entered it will display as the last cell.

### Screenshots
| Order Details | Refund Details |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/1062444/71740029-b0e74880-2e20-11ea-99ec-407d7841f50b.png" width="350" /> | <img src="https://user-images.githubusercontent.com/1062444/71740041-bcd30a80-2e20-11ea-9cd5-0271712ebf6a.png" width="350" /> |


Update release notes:
